specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -4138,175 +4319,9 @@
             "$ref": "#/parameters/SubscriptionId"
           },
           {
-            "name": "intentObjectName",
+            "name": "fabricName",
             "in": "path",
-            "description": "Replication protection intent name.",
-            "required": true,
-            "type": "string"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ReplicationProtectionIntent"
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Gets the details of a Replication protection intent item.": {
-            "$ref": "./examples/ReplicationProtectionIntents_Get.json"
-          }
-        }
-      },
-      "put": {
```

Comment:
Is this API removed?

Reply:
@lirenhe 
Can you please copy/paste some more context here (more line). Which API is this?
Github is not showing the diff as file is too large.
but we have not removed any API from swagger. We have introduced new version and have added few properties in some APIs

Reply:
@lirenhe : Any updates on this?